code/scripting/global_hooks.cpp

```diff
@@ -370,6 +370,17 @@ const std::shared_ptr<OverridableHook<>> OnHudCommMenuOpened = OverridableHook<>
 		{"Player", "object", "The player object."}
 	});
 
+const std::shared_ptr<OverridableHook<>> OnHudCommOrderIssued = OverridableHook<>::Factory("On HUD Comm Order Issued",
```

Comment:
So, I think this hook would massively benefit from having proper hook conditions.
Ship class / type / name of the target, same + eventual wing name for the recipient, maybe the sender.

Reply:
I'm not sure how to do conditions. What's a good one to look at for examples?

Reply:
No idea if I did it right, but a basic setup for conditions is there now.